recipe/build-mpi.sh

```diff
@@ -92,14 +101,18 @@ if [[ "$target_platform" == linux-* ]]; then
 fi
 
 ./configure --prefix=$PREFIX \
+            --enable-fast=O3 \
+            --enable-romio \
```

Comment:
Q: What do these do? How are they relevant to the UCX support?

Reply:
Apparently `--enable-romo` is the default, so removed that. `--enable-fast` sets the `O3` in compiler flags (including wrappers), where the default is O2. I don't believe it's relevant to UCX.